### ISSUE TITLE
How to finetune maskrcnn on HM3D

### ISSUE THREAD

**Issue by A-Nobel** [ajzhai/PEANUT#4]
Hello! I'm so sorry to bother you. I was hoping you could guide me on finetuning a COCO-pretrained Mask R-CNN using HM3D images. I am interested in exploring other semantic segmentation methods and would greatly appreciate it if you could recommend any available datasets or sample training pipelines. Thank you very much for your time and consideration!😸

**Comment by ajzhai** [comment]
Here is the data I used to finetune the Mask R-CNN: [https://uofi.box.com/s/kdsziq0nyicad3wlpuplaj8bc0okg3i9](https://uofi.box.com/s/kdsziq0nyicad3wlpuplaj8bc0okg3i9)

I used detectron2 to run Mask R-CNN. Here is an example training script: 
[train_seg.zip](https://github.com/ajzhai/PEANUT/files/13847360/train_seg.zip)

**Comment by A-Nobel** [comment]
Thank you so much for your assistance! The provided code and dataset work very well. I deeply appreciate your help and support!❤
